.lvversion

```diff
@@ -0,0 +1 @@
+20.0.1
```

Comment:
```suggestion
20.0
```

Reply:
This brings up a good question, I suppose. Does this work on LV 2020 without the SP1?

Reply:
From some quick searches, it sounds like LV 2020 had some issues. Since we also will be testing everything in LV 2020 SP1, I vote that we actually keep it as 20.0.1 like you have here.